### DISCUSSION TITLE
Safety of SEB, prevent cheating

### DISCUSSION THREAD

**Discussion by bnairolf** [SafeExamBrowser/seb-win-refactoring#963] [Q&A]
Hello,

I'm a simple teacher at school with not a big knowledge of programming. I want to use SEB in combination with a test on Moodle, but I haven't got still a lot of experience and information.

Firstly I wanted to know how safe the SEB is in combination with a test on Moodle. There are even explanations on youtube how to cheat with SEB, for example https://www.youtube.com/watch?v=mcYWrAM5UI8. Do they work? Is it easy for students with a little knowledge of programming to cheat? It seems that cheating is possible (https://github.com/SafeExamBrowser/seb-win-refactoring/discussions/732).

Secondly I wanted to ask what I can do as a simple teacher to prevent cheating: Is it enough to watch the students' screens if they don't open an other window than SEB? Or can they even cheat in a way that the window seems to be a window of SEB? Shall I forward the information of the discussion 833 (https://github.com/SafeExamBrowser/seb-win-refactoring/discussions/833) to our moodle administrator?

Thirdly: Shall we report possibilities of cheating that we find, so there can be searched for a solution?

Thank you very much in advance!

**Comment by dbuechel** [comment]
For enhanced security and monitoring (soon, that is with the upcoming version 3.8.0, also including screen proctoring) we strongly recommend the usage of SEB Server. If that is not possible, then you should use the Browser Exam Key (BEK) which ensures that students cannot enter an exam with a manipulated SEB version.

The VM circumvention shown above should not work anymore with the latest, official release versions of SEB for Windows. And forwarding the insights of #833 to your Moodle administrator is definitely a good idea!
